parameter_files/fates_params_default.cdl

```diff
@@ -591,6 +591,9 @@ variables:
 	double fates_fire_part_dens ;
 		fates_fire_part_dens:units = "kg/m2" ;
 		fates_fire_part_dens:long_name = "spitfire parameter, oven dry particle density, Table A1 Thonicke et al 2010" ;
+	double fates_fire_threshold ;
+		fates_fire_threshold:units = "kW/m" ;
+		fates_fire_threshold:long_name = "spitfire parameter, fire intensity threshold for tracking fires that spread" ;
```

Comment:
this fire_threshold parameter gives weird behavior if it is set to zero. as a minimum I suggest 0.001, or something not quite zero. that comment will go into documentation, but should we add a suggested range here?

Reply:
@rgknox including a zero value for fire_threshold here is not valid. Perhaps I should add a check in the code? thoughts on that.

Reply:
I think that sounds sensible.  I can put in a check that the number is greater than 0.001.
Also, this parameter is used in all contexts/scenarios when spitfire is turned on right? It is not used in a subset of spitifre that is not-necessarily active right?

Reply:
@rgknox you are correct. the fire_threshold is used in all scenarios where spitfire is on.

Reply:
@rgknox and thank you for adding a check. that will be a good fail safe.